src/scripts/validation.js

```diff
@@ -47,7 +47,7 @@ const disableButton = (buttonEl, config) => {
   buttonEl.classList.add(config.inactiveButtonClass);
```

Comment:
* There is an error in the console https://snipboard.io/2ZbBFL.jpg
* When I add a new card, there is an error https://snipboard.io/5rGWbK.jpg

Please, **export** the `disableButton` function from `validation.js`, to be able to use it in `index.js`


```js
export const disableButton = (buttonEl, config) => {
  buttonEl.disabled = true;
  buttonEl.classList.add(config.inactiveButtonClass);
};


```